tests/common/snappi/common_helpers.py

```diff
@@ -684,3 +684,24 @@ def get_ipv6_addrs_in_subnet(subnet, number_of_ip):
 def sec_to_nanosec(secs):
     """ Convert seconds to nanoseconds """
     return secs * 1e9
+
+
+def get_pfc_pause_frame_count(duthost, port, priority, is_Tx=False):
```

Comment:
It seems to me that get_pfc_count is a better name because SONiC cannot really differentiate PFC pause and resume frames. @neethajohn  Correct me if I am wrong.

Reply:
Good catch, will rename